guides/common/modules/proc_disable-transparent-huge-pages.adoc

```diff
@@ -0,0 +1,11 @@
+[id="Disable_Transparent_Huge_Pages_{context}"]
+= Disable Transparent Huge Pages
+
+Transparent Huge Pages is a memory management technique used by the Linux kernel to reduce the overhead of using the Translation Lookaside Buffer (TLB) by using larger sized memory pages.
+Due to databases having Sparse Memory Access patterns instead of Contiguous Memory access patterns, database workloads often perform poorly when Transparent Huge Pages is enabled.
```

Comment:
I know you're just moving things around here, but I wonder if `Transparent Huge Pages` is singular or plural. Pages makes me think it's plural, but it's just a single feature so I'm not sure.

Reply:
"Transparent Huge Pages" (including quotes) giving 42200 hits on Google, "Transparent Huge Page" is 8220, but first is link to https://docs.kernel.org/admin-guide/mm/transhuge.html which says "Transparent Hugepage", so I'll rename.

Reply:
Well, I wonder if it should be `Transparent Huge Pages are a ...`, but here I really don't know because I'm not a native speaker.